recipes-app/node-red/files/node-red.service.tmpl

```diff
@@ -4,7 +4,9 @@ After=syslog.target network.target
 
 [Service]
 Type=idle
-ExecStart=/usr/bin/node-red -u %h/.node-red
+User=nodered
+WorkingDirectory=${NODE_RED_HOME_DIR}
+ExecStart=/usr/bin/node-red -u ./.node-red
```

Comment:
Why do we need to specify the userdir on node-red start? Doesn't `$HOME/.node-red` work anymore?

Reply:
I'm not a node-red expert. According to the docs, the default is indeed `$HOME/.node-red`. By that, we could remove the `-u ...`. Don't know which other implications this has. https://nodered.org/docs/getting-started/local